JavaOOPHomework1/src/Computer.java

```diff
@@ -0,0 +1,22 @@
+import java.time.LocalDate;
+
+/**
+ * base class for electronics, holds guarantee period
+ */
+public class Computer extends ElectronicProduct {
+
+    public Computer(String name, double price, int quantity, String ageRestriction, LocalDate guaranteePeriod) {
```

Comment:
comentariul legat de guaranteePeriod este valabil si in acest caz, cu mentiunea ca trebuie setat cu valoarea 24